src/EditorFeatures/Core/LanguageServer/RazorInProcLanguageClient.cs

```diff
@@ -60,6 +60,13 @@ internal class RazorInProcLanguageClient : AbstractInProcLanguageClient
             _defaultCapabilitiesProvider = defaultCapabilitiesProvider;
         }
 
+        protected override void Activate_OffUIThread()
```

Comment:
oh, we may want to do this in liveshare too, since it does use the default capabilities as well

Reply:
I decided to do the minimum, since the razor client is the one causing the UI delay, and not tempt the RPS gods. Will follow up if you feel strongly, but going to merge this and see what happens with the insertion